content/en/catalyst/fund9/challenges/products-and-integrations/campagin-brief.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Products and Integrations"
```

Comment:
Remove this challenge, because it had been merged with 'Dapps, Product & Integrations' challenge.